app/subcommands/group/run

```diff
@@ -6,14 +6,16 @@ set -eu
 
 [ -n "${1:-}" ] || fatality 'must provide a group name to add too'
 group_name="$1"
+client_variant="${2:-}"
```

Comment:
doing this method would limit usage to one argument. Instead you can do a `shift` here and down on line `21` add `"$@"` instead of this variable name to pass any and all left over variables. Would need to do another `shift` and `"$@"` inside `run_with_dependencies`.